draft-ietf-scitt-architecture.md

```diff
@@ -74,7 +74,25 @@ normative:
     date: 2022-07-22
   DID-WEB:
     target: https://w3c-ccg.github.io/did-method-web/
-    title: did:web Decentralized Identifiers Method Spec
+    title:  did:web Decentralized Identifiers Method Spec
+  SPDX:
+    target: https://spdx.dev/use/specifications/
+    title:  SPDX Specification
+  SWID:
+    target: https://csrc.nist.gov/projects/software-identification-swid/
```

Comment:
I will venture a guess that you can be more specific if you point to either.

The NIST specifications:

```suggestion
    target: https://csrc.nist.gov/Projects/Software-Identification-SWID/guidelines
```

Or the referenced ISO specifications, which is what NIST recommends you do on the changed link above.

> The SWID Tag format, defined by the International Organization for Standardization (ISO) and the International Electrotechnical Commission (IEC) standard ISO/IEC 19770-2, is a structured metadata format for describing a software product. NIST recommends use of the latest version of this standard, [ISO/IEC 19770-2:2015](https://webstore.ansi.org/RecordDetail.aspx?sku=ISO%2fIEC+19770-2%3a2015).

```suggestion
    target: https://www.iso.org/standard/65666.html
```